nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/ConsumeAzureEventHub.java

```diff
@@ -86,7 +86,10 @@
 import static org.apache.commons.lang3.StringUtils.defaultIfBlank;
 
 @Tags({"azure", "microsoft", "cloud", "eventhub", "events", "streaming", "streams"})
-@CapabilityDescription("Receives messages from Azure Event Hubs, writing the contents of the message to the content of the FlowFile.")
+@CapabilityDescription("Receives messages from Microsoft Azure Event Hubs, writing the contents of the message to the content of the FlowFile. "
+        + "This processor shares the same functionality as GetAzureEventHub but supports checkpoints via Azure Blob Storage. "
```

Comment:
As the preferred approach, the initial description should not be defined in terms of similarity to `GetAzureEventHub`. Recommend the following wording for the first sentence:
```suggestion
        + "Receives messages from Microsoft Azure Event Hubs with checkpointing to ensure consistent event processing. "
```